docs/locale/ru_RU/README.md

```diff
@@ -0,0 +1,33 @@
+# Fabric Docs ru_RU
+
+A project to translate the Fabric documentation into Russian.
+
+Проект по переводу документации Fabric на русский.
+
+# Scope of Lab
+
+The purpose of this lab is to translate the Fabric documentation into Russian. We will use this repository to sync with the latest English version of Hyperledger Fabric, starting with version 2.2.
+
+For subsequent releases, any update to source files will be translated in a new release base branch. Issues are used to track what to update in the changed docs.
+
+Цель этой лаборатории - перевод документации Fabric на русский. Мы будем использовать этот репозиторий чтобы синхронизироваться с последней английской версией Hyperledger Fabric, начиная с версии 2.2.
+
+Для будущих релизов, любое обновление исходных файлов будет переведено в новом релизе основной ветки. Issues используются, чтобы отслеживать что будет обновлено в измененных доках.
+
+The original online documentation is: http://hyperledger-fabric.readthedocs.io/
+
+The Brazilian Portuguese online documentation is: coming soon...
+
+Оригинальная онлай-документация：http://hyperledger-fabric.readthedocs.io/
+
+Русская онлайн документация : скоро будет...
+
+# Initial committers - мейнтейнеры
+
+* Arseniy Chekanov - senya@chekanov.net
+
+# Want to contribute? - Хочешь поконтрибутить?
```

Comment:
A small nit, but I would rather change this to:

"Хочешь поучавствовать?"

the goal  is to have documentation translate rather than transliterate English words into Russian.